cmd/kubeadm/app/phases/upgrade/postupgrade.go

```diff
@@ -251,3 +253,28 @@ func rollbackFiles(files map[string]string, originalErr error) error {
 	}
 	return errors.Errorf("couldn't move these files: %v. Got errors: %v", files, errorsutil.NewAggregate(errs))
 }
+
+// LabelMasterNodes finds all nodes with the "master" node-role label and also applies
+// the "control-plane" node-role label to them.
+// TODO: https://github.com/kubernetes/kubeadm/issues/2200
+func LabelMasterNodes(client clientset.Interface) error {
```

Comment:
this function is temporary until, the master node label is removed.

Reply:
might be worth making this explicit in the code

Reply:
left it like so, it's "business logic" and having it in "apply" doesn't feel right, even if kubeadm is messy in this regard.